impeller/renderer/command.cc

```diff
@@ -58,11 +58,11 @@ bool Command::DoBindResource(ShaderStage stage,
   switch (stage) {
     case ShaderStage::kVertex:
       vertex_bindings.buffers[slot.ext_res_0] = {
-          .slot = slot, .view = BufferResource(metadata, view)};
+          .slot = slot, .view = BufferResource(metadata, std::move(view))};
       return true;
     case ShaderStage::kFragment:
       fragment_bindings.buffers[slot.ext_res_0] = {
-          .slot = slot, .view = BufferResource(metadata, view)};
+          .slot = slot, .view = BufferResource(metadata, std::move(view))};
```

Comment:
If you accidentally omit the std::move here, we get a linter failure right?

Reply:
We do not :(